routing/following_info.hpp

```diff
@@ -73,6 +73,8 @@ class FollowingInfo
   std::string m_targetName;
   // Street name to display. May be empty.
   std::string m_displayedStreetName;
```

Comment:
I see one strange thing here (except the vars' names - this is a separate issue).

AFAI understand, m_displayedStreetName == m_targetName with some processing?
We use m_displayedStreetName (not m_targetName) constructing android.RoutingInfo
Should we do the same for m_nextStreetName?